boards/xtensa/esp32_ethernet_kit/board_init.c

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2022 Grant Ramsay <grant.ramsay@hotmail.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/drivers/gpio.h>
+
+#define IP101GRI_RESET_N_PIN	5
+
+static int board_esp32_ethernet_kit_init(const struct device *dev)
+{
+	ARG_UNUSED(dev);
+	const struct device *gpio = DEVICE_DT_GET(DT_NODELABEL(gpio0));
+
+	if (!device_is_ready(gpio)) {
+		return -ENODEV;
+	}
+
+	/* Enable the Ethernet phy */
+	gpio_pin_configure(gpio, IP101GRI_RESET_N_PIN, GPIO_OUTPUT);
```

Comment:
what would happen if those gpio configuration fuctions fail? 

I suggest to track when they fail, otherwise it would be hard to debug if external phy fails because of something related to reset.